src/errorpage.cc

```diff
@@ -711,6 +723,7 @@ ErrorState::ErrorState(HttpRequest * req, HttpReply *errorReply, const AccessLog
     Must(errorReply);
     response_ = errorReply;
     httpStatus = errorReply->sline.status();
+    debugs(4, 3, "relaying " << *this);
```

Comment:
We could merge the two added debugs() lines above into the shared (private) ErrorState constructor instead, but I do not think we should because

* it is useful to distinguish the two ErrorState construction cases; and
* the shared constructor does not have access to HTTP status code information unless we add more code).
 
The unique (within ErrorState constructor debugging contexts) "relaying" prefix helps with the "distinguish" part (without having to map line numbers in cache.log messages to version-dependent Squid code).